src/data/emacs-mode/agda-input.el

```diff
@@ -213,9 +213,9 @@ order for the change to take effect."
 
   ;; Inequality and similar symbols.
 
-  ("leq"  . ,(agda-input-to-string-list "<≪⋘≤≦≲ ≶≺≼≾⊂⊆ ⋐⊏⊑ ⊰⊲⊴⋖⋚⋜⋞＜"))
+  ("leq"  . ,(agda-input-to-string-list "<≪⋘≤≦≲ ≶≺≼≾⊂⊆⫇⫉ ⋐⊏⊑ ⊰⊲⊴⋖⋚⋜⋞＜"))
   ("leqn" . ,(agda-input-to-string-list "≮  ≰≨≴⋦≸⊀ ⋨⊄⊈⊊  ⋢⋤ ⋪⋬   ⋠"))
-  ("geq"  . ,(agda-input-to-string-list ">≫⋙≥≧≳ ≷≻≽≿⊃⊇ ⋑⊐⊒ ⊱⊳⊵⋗⋛⋝⋟＞"))
+  ("geq"  . ,(agda-input-to-string-list ">≫⋙≥≧≳ ≷≻≽≿⊃⊇⫈⫊ ⋑⊐⊒ ⊱⊳⊵⋗⋛⋝⋟＞"))
```

Comment:
In general, inserting things in the middle here is not backwards-compatible.  Folks might have learned to input `\geq` followed by a number to get directly to their symbol.
On the other hand we want symbols in a logical order; so the symbols are inserted in the correct place here.
Dunno.

Reply:
One possible solution is to split off the `agda-input.el` so that it is versioned independently. Then, we can add as many symbols as possible in each release, and people who don’t want new symbols can stick with the version they like.

Reply:
`agda-input.el` is very useful even without agda, so IMO, it shouldn’t be coupled with Agda.